k8s/namespaces/admin/aad-pod-identity/aad-pod-identity.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  labels:
+    k8s-app: aad-pod-id
+  name: aad-pod-identity
+  namespace: admin
+spec:
+  releaseName: aad-pod-identity
+  chart:
+    repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
+    name: aad-pod-identity
+    version: 1.6.2
+  values:
+    installMICException: false
```

shouldn't this be true?
```suggestion
    installMICException: true
```

---

Fine to be corrected - but this is what that configures - https://github.com/Azure/aad-pod-identity/blob/v1.6.2/charts/aad-pod-identity/templates/mic-exception.yaml - which I couldn't find in our current setup

---

https://github.com/Azure/aad-pod-identity#1-deploy-aad-pod-identity

> # For AKS clusters, deploy the MIC and AKS add-on exception by running -
kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/mic-exception.yaml

---

Cool thanks, I've removed it as it's true by default